### DISCUSSION TITLE
New place to discuss bugs, suggestions and ideas

### DISCUSSION THREAD

**Discussion by natsabari** [spacescan-io/Spacescan-discussion#65] [Announcements]
Excited to try this new format to Manage and support Chia community on thinks related to Spcescan bugs, suggestions and Ideas.

Welcome you all to try out this new format and hope it would be better that a long  threads in discord.

**Comment by MarvinQuevedo** [comment]
I think that could be interesting show the memos in the transaction page

**Reply by natsabari** [reply]
Got it, will create ticket to track it
